form.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable require-jsdoc */
+const formName = document.querySelector('#name');
+const email = document.querySelector('#email');
+const formtext = document.querySelector('#text');
+const inputData = {};
+const storage = window.localStorage;
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+      e.code === 22 ||
+      e.code === 1014 ||
+      e.name === 'QuotaExceededError' ||
+      e.name === 'ERROR_QUOTA_REACHED') && (storage && storage.length !== 0);
+  }
+}
+
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure that the form data for all inputs should be stored in a single object, and saved to a single local storage registry.

```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```